patches/amd-mainline/hipBLASLt/0008-tensilelite-make-roc-smi-binary-optional-during-buil.patch

```diff
@@ -0,0 +1,28 @@
+From e0754df413a35467e9c3b19a084b95899a371186 Mon Sep 17 00:00:00 2001
+From: Scott Tsai <scottt.tw@gmail.com>
+Date: Tue, 8 Apr 2025 04:28:51 +0800
+Subject: [PATCH 8/8] tensilelite: make roc-smi binary optional during build
+
+---
+ tensilelite/Tensile/Common/GlobalParameters.py | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/tensilelite/Tensile/Common/GlobalParameters.py b/tensilelite/Tensile/Common/GlobalParameters.py
+index 8ec5f326..6045e2af 100644
+--- a/tensilelite/Tensile/Common/GlobalParameters.py
++++ b/tensilelite/Tensile/Common/GlobalParameters.py
+@@ -549,7 +549,10 @@ def assignGlobalParameters(config, isaInfoMap: Dict[IsaVersion, IsaInfo]):
+         globalParameters["CmakeCCompiler"] = os.environ.get("CMAKE_C_COMPILER")
+ 
+     globalParameters["ROCmBinPath"] = os.path.join(globalParameters["ROCmPath"], "bin")
+-    globalParameters["ROCmSMIPath"] = locateExe(globalParameters["ROCmBinPath"], "rocm-smi")
++    try:
++        globalParameters["ROCmSMIPath"] = locateExe(globalParameters["ROCmBinPath"], "rocm-smi")
++    except OSError:
++        pass
```

Comment:
FYI, this works around #359.

Reply:
@marbre , since `rocm-smi` would not work under the WSL for some time(?), should we ask hipBLASLt to integrate this change?

Reply:
We'll address this in a meeting later today.